schema_based_on_diagram.sql

```diff
@@ -0,0 +1,35 @@
+-- Create a database based on a diagram
+create table medical_histories(id int generated always as identity primary key,
+admitted_at timestamp,
+patient_id int references patients(id),
+status varchar(70));
+
+ create table patients(id int generated always as identity primary key,
+ name varchar(300),
+ date_of_birth date);
+
+ create table invoices(id int generated always as identity primary key,
+total_amount decimal,
+generated_at timestamp,
+pay_at timestamp,
+medical_history_id int references medical_histories(id));
+
+create table invoice_items(id int generated always as identity primary key,
+unit_price decimal,
+quantity int,
+total_price decimal,
+invoice_id int references invoices(id),
+treatment_id int references treatments(id));
+
+create table treatments(id int generated always as identity primary key,
+type varchar(500),
+name varchar(300));
```

Comment:
- [ ] Please add the join the table between `medical_histories` and `treatments` tables as there is a many to many relation between those tables. Also, please make sure to create an index for that join table.